Makefile.rhpam

```diff
@@ -0,0 +1,85 @@
+include Makefile
+
+
+VERSION = 7.11.0
+CHANNELS = 7.x
+DEFAULT_CHANNEL = 7.x
+BUNDLE_CHANNELS = --channels=$(CHANNELS)
+BUNDLE_DEFAULT_CHANNEL = --default-channel=$(DEFAULT_CHANNEL)
+
+IMAGE_TAG_BASE = registry.stage.redhat.io/rhpam-7/rhpam-kogito-rhel8-operator
```

Comment:
use only rhpam-7/rhpam-kogito-rhel8-operator here.
and where this env is used?

Reply:
This env is used in Makefile https://github.com/vaibhavjainwiz/kogito-operator/blob/c1f48b775716ceccad46bff934e62d8e0b1b499b/Makefile#L38

Reply:
changed `IMAGE_TAG_BASE` value to rhpam-7/rhpam-kogito-rhel8-operator